h/views.py

```diff
@@ -7,10 +7,13 @@
 from pyramid.events import ContextFound
 from pyramid.view import forbidden_view_config, notfound_view_config
 from pyramid.view import view_config
+import pyramid.response
 
 from . import session
 from .models import Annotation
 from .resources import Application, Stream
+import h.api_client.api_client as api_client
```

Comment:
So, actually, looking more carefully, 2. applies to the `h.atom_feed` import, but this line is actually an unused import, I think?